internal/models/rsvp.go

```diff
@@ -13,15 +13,20 @@ var (
 )
 
 const RSVPExpiryTime = 3 * 24 * time.Hour
+const RSVPExpiryDate = 1608353999000 // this is milliseconds since epoch for the date 12/18/2020 23:59:59 EST
```

Comment:
Instead of dealing with numbers it's usually a lot easier to deal with go's `time.Time`
So this would use https://golang.org/pkg/time/#Date and would look like `time.Date(2020, time.December, 18, 23, 59, 59, 0, time.EST)`

Then, instead of checking `(time.Now().UnixNano()/1000000) > models.RSVPExpiryDate` you can use https://golang.org/pkg/time/#Time.After and say `if time.Now().After(RSVPExpiryDate)`

Reply:
Sounds good. Want to mention that we won't be able to use RSVPExpiryDate as a const anymore, it's going to be a var.